lesson7giang/src/main/java/com/example/lesson7giang/AddNewStudent.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lesson7giang;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.EditText;
+
+public class AddNewStudent extends AppCompatActivity {
+    public static String EXTRA_USER_NAME = "NAME";
+    public static String EXTRA_USER_AGE = "AGE";
```

Comment:
EXTRA nên có thêm cả tên package
```suggestion
    public static String EXTRA_USER_AGE = "com.example.lesson7giang.AGE";
```